roles/rsyslog/roles/input_roles/viaq/tasks/main.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 - name: Set Viaq facts
   set_fact:
-    rsyslog_role_packages: "{{ rsyslog_viaq_prereq_packages|flatten|d([]) }}+{{ rsyslog_viaq_packages|flatten|d([]) }}"
+    rsyslog_role_packages: "{{ rsyslog_viaq_prereq_packages + rsyslog_viaq_packages }}"
```

Comment:
see above

Reply:
#87 (comment) is applied here, as well.

Reply:
Note: var name `rsyslog_role_packages` was renamed to `__rsyslog_packages` and the var is set with instead of `set_fact` but in `var:`.